astropy/table/column.py

```diff
@@ -292,6 +291,13 @@ def copy(self, order='C', data=None, copy_data=True):
 
         out = data.view(self.__class__)
         out.__array_finalize__(self)
+
+        # If there is meta on the original column then deepcopy (since "copy" of column
+        # implies complete independence from original).  __array_finalize__ will have already
+        # made a light copy.  I'm not sure how to avoid that initial light copy.
+        if self.meta is not None:
+            out.meta = self.meta  # MetaData descriptor does a deepcopy here
```

Comment:
This seems a bit strange. Shouldn't we use `MetaData(copy=False)` for `Column` as well, if we are using it for `Table`?